drivers/hmis/app/jobs/hmis/candidate_pool_builder_job.rb

```diff
@@ -0,0 +1,28 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Hmis
+  class CandidatePoolBuilderJob < BaseJob
+    include NotifierConfig
+
```

Comment:
Maybe @eanders can confirm if it's best practice, but if you add
```
queue_as ENV.fetch('DJ_LONG_QUEUE_NAME', :long_running)
```
It should default to queueing as long-running, which I think we want.